tests/functional/spec/features/repeating_sections/repeating_sections_with_hub_and_spoke.js

```diff
@@ -252,7 +252,7 @@ describe('Feature: Repeating Sections with Hub and Spoke', function () {
         .setValue(DateOfBirthPage.year(), '1990')
         .click(DateOfBirthPage.submit())
 
-        .getText(ConfirmDateOfBirthPage.questionText()).should.eventually.equal('Samuel Clemens is 28 years old. Is this correct?')
+        .getText(ConfirmDateOfBirthPage.questionText()).should.eventually.contain('Samuel Clemens is')
```

Comment:
I think this line should be removed completely - We shouldn't care what the text is midway through a test and the test will fail later if "Samuel's" name isn't present.

Reply:
I have removed this line.